src/common/tusb_fifo.c

```diff
@@ -132,8 +132,8 @@ bool tu_fifo_read(tu_fifo_t* f, void * buffer)
 
 /******************************************************************************/
 /*!
-    @brief This function will read n elements into the array index specified by
-    the write pointer and increment the write index. If the write index
+    @brief This function will read n elements from the array index specified by
```

Comment:
Since this comment is updated anyway, it could be changed, **n** in description can easily be confused with **n** in function name tu_fifo_read_**n** and they are not related. It it was **count** instead of n confusion would be gone.

Reply:
right, this happens when I try to write the function descriptions before its implementation. :D

Reply:
Could you take a look also other commets ? English not my native language ;)